README.rst

```diff
@@ -3,3 +3,7 @@ Semaphore
 #########
 
 Semaphore is the user messaging service for the Rubin Science Platform.
+
+==================
+
+Summaries will be interpreted using the first provided body paragraph unless already defined in the `summary` property in the post metadata
```

Comment:
Whoops. I actually meant to update the README in https://github.com/lsst-sqre/rsp_broadcast. We can drop this paragraph from here.